src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -1846,14 +1846,14 @@ private function addRateLimiterSection(ArrayNodeDefinition $rootNode)
                                     ->enumNode('strategy')
                                         ->info('The rate limiting algorithm to use for this rate')
                                         ->isRequired()
-                                        ->values(['fixed_window', 'token_bucket', 'sliding_window'])
+                                        ->values(['fixed_window', 'token_bucket', 'sliding_window', 'no_limit'])
                                     ->end()
                                     ->integerNode('limit')
                                         ->info('The maximum allowed hits in a fixed interval or burst')
                                         ->isRequired()
                                     ->end()
                                     ->scalarNode('interval')
-                                        ->info('Configures the fixed interval if "strategy" is set to "fixed_window" or "sliding_window". The value must be a number followed by "second", "minute", "hour", "day", "week" or "month" (or their plural equivalent).')
+                                        ->info('Configures the fixed interval if "strategy" is set to "fixed_window", "sliding_window" or "no_limit". The value must be a number followed by "second", "minute", "hour", "day", "week" or "month" (or their plural equivalent).')
```

Comment:
This is not true, `no_limit` doesn't have any options (as it blindly accepts everything you throw at it)

Reply:
Oops. Sorry